lib/src/test/kotlin/starknet/provider/ProviderTest.kt

```diff
@@ -411,4 +411,31 @@ class ProviderTest {
             ContractDefinition("{}")
         }
     }
+
+    @ParameterizedTest
+    @MethodSource("getProviders")
+    fun `estimate fee`(provider: Provider) {
+        // FIXME: Not yet supported in Devnet RPC
+        if (provider is JsonRpcProvider) {
+            return
+        }
```

Comment:
I believe should be supporting fee estimation. If it's not working for you this could be a bug either in devnet or in this pr.